public/resources/functions/helpers.private.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable camelcase */
+// Temporary disabling camelcase rule. This require a change in the plugin code
+
+const moment = require('moment-timezone');
+
+function handleError(error) {
+  let message = '';
+  if (error.message) {
+    message += error.message;
+  }
+  if (error.stack) {
+    message += ` | stack: ${error.stack}`;
+  }
+  (console.error || console.log).call(console, message || error);
+}
+
+/**
+ * Get a Task Resource
+ *
+ * @param {object} context Twilio function context object
+ * @param {string} sid Call Sid or Task Sid
+ * @returns {Promise} Promise Object with Task Resource
+ */
+function getTask(context, sid) {
+  const client = context.getTwilioClient();
+  let fetchTask;
+
+  if (sid.startsWith('CA')) {
+    fetchTask = client.taskrouter.workspaces(context.TWILIO_WORKSPACE_SID).tasks.list({
+      evaluateTaskAttributes: `call_sid= '${sid}'`,
+      limit: 20,
+    });
+  } else {
+    fetchTask = client.taskrouter.workspaces(context.TWILIO_WORKSPACE_SID).tasks(sid).fetch();
+  }
+
+  return fetchTask
+    .then((result) => {
+      const task = Array.isArray(result) ? result[0] : result;
+      res = {
+        status: 'success',
+        topic: 'getTask',
+        action: 'getTask',
+        taskSid: task.sid,
+        taskQueueSid: task.taskQueueSid,
+        taskQueueName: task.taskQueueFriendlyName,
+        workflowSid: task.workflowSid,
+        workspaceSid: task.workspaceSid,
+        data: task,
+      };
+      return res;
+    })
+    .catch((error) => {
+      res = {
+        status: 'error',
+        topic: 'getTask',
+        action: 'getTask',
+        data: error,
+      };
+      return res;
+    });
+}
+
+async function cancelTask(client, workspaceSid, taskSid) {
```

Comment:
Would be nice to have a js docblock for these methods as well!

Reply:
Solved in e98d19b